ratpack-retrofit2/src/main/java/ratpack/retrofit/RatpackRetrofitCallException.java

```diff
@@ -57,7 +58,11 @@ public RatpackRetrofitCallException(Call<?> call, String error) {
    * @since 1.6.0
    */
   public RatpackRetrofitCallException(Request request, Response<?> response) {
-    super(request.url().toString() + ": " + Exceptions.uncheck(() -> response.errorBody().string()));
+    super(request.url().toString() + ": " + Exceptions.uncheck(() -> {
+      Charset charset = response.errorBody().contentType() == null ? null : response.errorBody().contentType().charset();
+      if (charset == null) charset = Charset.forName("UTF-8");
```

Comment:
You will need to use `{}` here to wrap the body of the `if` statement in order to comply with the style check in this project.

Reply:
Done!